flowmachine/flowmachine/core/sqlalchemy_utils.py

```diff
@@ -0,0 +1,133 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import pandas as pd
+from sqlalchemy import Table, MetaData
+from sqlalchemy.dialects import postgresql
+from sqlalchemy.sql import Selectable
+
+
+def get_sqlalchemy_table_definition(fully_qualified_table_name, *, engine):
```

Comment:
I am not sure the name is really explanatory here. I first thought you were getting a generic table when you are only after `events` table. Second, I would rather move this to `event_table_subset.py` unless you consider using it in other places.

Reply:
Yes, good point. The name was much more general than the implementation. I have changed the implementation so that it will work for any existing table. I do expect to reuse this in other places, so I'd rather keep it here.